_guidelines-v4/01-introduction/01-about/02-acknowledgments.md

```diff
@@ -0,0 +1,19 @@
+---
+version: "v4"
+title: "Acknowledgments"
+sectionid: "acknowledgments"
+---
+
+Many institutions and individuals assisted in the preparation of these Guidelines and in the overall development of the Music Encoding Initiative framework.
+
+Grateful acknowledgment is given to the following institutions for their generous contributions: the Akademie der Wissenschaften und der Literatur (AdW) in Mainz for serving as hosting institution for the MEI Community, and the National Endowment for the Humanities (NEH) and the Deutsche Forschungsgemeinschaft (DFG) for their joint financial support of the MEI project in its early stages. We thank several institutions that hosted Music Encoding Conferences or other MEI-related meetings in the past: The AdW Mainz, the University of Virginia Library, the Biblioteca Umanistica of the Università degli Studi Firenze, McGill University Montréal, the Centre d’études supérieures de la Renaissance Tours, the Maryland Institute for Technology in the Humanities (MITH) in College Park, the Oxford e-Research Centre, the Universität Paderborn and the Österreichische Akademie der Wissenschaften Wien. We also thank all other institutions that allow their researchers to invest time into both the community and the encoding framework. It is their interest that makes MEI an incredible platform for interchange and scholarly progress.
+
+The Text Encoding Initiative is also owed a special debt of gratitude. In addition to providing much of the inspiration for MEI, the TEI organization supplied funding for the MEI Technical Group in its efforts to adopt ODD. The editors of these Guidelines are grateful for those of the TEI, which provided a stellar exemplar and from which we have borrowed shamelessly.
```

Comment:
…grateful ~~for~~ to those of the TEI, ~~which~~ who…

Reply:
this refers to the TEI Guidelines, not the editors of the TEI Guidelines…